environment-dev.yml

```
@@ -34,6 +34,7 @@ dependencies:
  - ecmwf-api-client
  - cxx-compiler
  - sysroot_linux-64
+ - rasterio
```

I have a suggested version of this that won't require rasterio but will require numba. 
```suggestion
 - numba
```